modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/CPDefinitionSystemObjectRelatedModelsProviderTest.java

```diff
@@ -30,6 +47,79 @@
 public class CPDefinitionSystemObjectRelatedModelsProviderTest
 	extends BaseSystemObjectRelatedModelsProviderTestCase {
 
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_objectDefinition = ObjectDefinitionTestUtil.publishObjectDefinition();
+
+		_systemObjectDefinition = getSystemObjectDefinition();
+
+		ObjectField objectField = ObjectFieldLocalServiceUtil.getObjectField(
+			_systemObjectDefinition.getObjectDefinitionId(), "name");
+
+		ObjectDefinitionLocalServiceUtil.updateTitleObjectFieldId(
+			_systemObjectDefinition.getObjectDefinitionId(),
+			objectField.getObjectFieldId());
```

Comment:
@rijema the title field is already the name, do we need to update this again in this test?

![image](https://github.com/user-attachments/assets/7080ea9c-e12b-4cbc-9122-3bac6e099dc4)

Reply:
Removed.

Reply:
Just a comment: My first thought was to consider to test the other fields such description and shortDescription as well,that's why I was considering the title field. Yet, the test was turning longer than I expected.